Apps/Sandcastle/gallery/development/3D Models Instancing.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Create 3D models using glTF.">
+    <meta name="cesium-sandcastle-labels" content="Development">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.9/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    "use strict";
+//Sandcastle_Begin
+
+var debugShowBoundingVolume = false;
+var debugWireframe = false;
+var globeVisible = false;
+
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    globe : globeVisible,
+    skyAtmosphere : globeVisible
+});
+
+var scene = viewer.scene;
+
+function createCollection(url, count) {
+    var instances = [];
+    var longitude = -123.0744619;
+    var latitude = 44.0503706;
+    var height = 5000.0;
+    for (var i = 0; i < count; ++i) {
+        var heading = Math.random();
+        var pitch = Math.random();
+        var roll = Math.random();
+        var position = new Cesium.Cartesian3.fromDegrees(longitude, latitude, height + i * 20);
+
+        var modelMatrix = Cesium.Transforms.headingPitchRollToFixedFrame(position, heading, pitch, roll);
+        instances.push({
```

Comment:
We'll need an explicit `ModelInstance` type as we flush out picking, show/hide, etc.